src/components/calcite-dropdown-item/calcite-dropdown-item.tsx

```diff
@@ -53,6 +53,13 @@ export class CalciteDropdownItem {
   /** pass an optional title for rendered href */
   @Prop() linkTitle?: string;
 
+  /** optionally pass an icon to display - accepts calcite ui icon names  */
+  @Prop({ reflect: true }) icon?: string;
+
+  /** optionally used with icon, select where to position the icon */
+  @Prop({ reflect: true, mutable: true }) iconPosition?: "start" | "end" =
```

Comment:
We've been using "leading" | "trailing" but this works too :)

Reply:
😬 We have "start" / "end" in a couple other components that use `icon-position`. Would be nice to standardize one way or the other here in our beta wonderland, lol.

Reply:
yeah. It would be nice if we can standardize either using `position:start/end` or `layout:leading/trailing`. We can change ours, it would just be a breaking change.

Reply:
`start/end` matches the [logical properties spec](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Logical_Properties) better, IMO